src/tool/subcommands/archive_cmd.rs

```diff
@@ -862,8 +899,13 @@ async fn sync_bucket(
     let store = Arc::new(ManyCar::try_from(snapshot_files)?);
     let heaviest_tipset = store.heaviest_tipset()?;
 
-    let info =
-        ArchiveInfo::from_store(&store, "ManyCAR".to_string(), heaviest_tipset.clone(), None)?;
+    let info = ArchiveInfo::from_store(
+        &store,
+        "ManyCAR".to_string(),
+        heaviest_tipset.clone(),
+        1,
```

Comment:
It's difficult to say what this `1` means without going deeper into the implementation. It'd be nice to have it named.

Reply:
Fixed